### ISSUE TITLE
Redirect all freecad owned domains (freecadweb.org, freecad.org) to freecad.org

### ISSUE THREAD

**Issue by memtech3** [FreeCAD/FreeCAD-Homepage#119]
Have y'all considered redirecting all freecad owned domains (freecadweb.org, freecad.org) to freecad.org? By this I mean putting a http 301 permanent redirect on the freecadweb.org domain

**Comment by yorikvanhavre** [comment]
Yes that's on out todo list... We hoped to find a way to keep both domains but most webapps we use (phpbb and mediawiki basically) require one domain.
First we need to finish migrating (and testing) these two apps to use the freecad.org domain, then we see what we do about freecadweb.org...

**Comment by memtech3** [comment]
I think it looks more professional to keep it to one domain with the other one set as a redirect

**Comment by hasecilu** [comment]
The tracker link (http://www.freecad.org/tracker/) redirects to Mantis, so it would be better to redirect to GitHub issues, used on forum.

**Comment by chennes** [comment]
I've updated Gandi to forward tracker.freecad.org to the GitHub issues page. @kkremitzki it looks like there is a DNS A record for "tracker" that we probably no longer need, but I left it alone.

**Comment by luzpaz** [comment]
Can we close this ?
